Parser/asdl.py

```diff
@@ -33,8 +33,7 @@
 # See the EBNF at the top of the file to understand the logical connection
 # between the various node types.
 
-builtin_types = {'identifier', 'string', 'bytes', 'int', 'object', 'singleton',
-                 'constant'}
+builtin_types = {'identifier', 'string', 'int', 'constant'}
```

Comment:
Update also a comment in Parser/Python.asdl.

I am wondering, should we keep `constant` or `object`?

Reply:
neither of them are defined in ASDL spec, but I guess `constant` is more explicit about its job then `object`. So I'd go for keeping it as is, and using `constant`

Reply:
In any case we keep `ast2obj_object()` and `obj2ast_object()`.

Reply:
Yes, because others are a symlink to it. Do you want me to change constant with object, or is it OK this way?